### PR TITLE
updates filer clear sizes on popup

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:
- updates filer clear sizes on popup

- [x] responsive behaviour tested
- [x] browser and device tested
- [x] passes validation (html / bootstrap)
- [x] SEO optimized (html outline)
- [x] printability (print css)
- [x] doesn’t break anything else

### CODE REVIEW

File: djangocms_admin_style/sass/components/_popup.scss [hunk 1]
```diff
@@ -18,6 +18,12 @@
 }
 .popup,
 .cms-admin-modal {
+    &.app-aldryn_bootstrap3 {
+        .filerFile .filerClearer {
+            width: 36px !important;
+            height: 36px !important;
```
Comment (Flight) [inline djangocms_admin_style/sass/components/_popup.scss]: Do we really need !important here?

File: djangocms_admin_style/sass/components/_popup.scss [hunk 2]
```diff
@@ -18,6 +18,12 @@
 }
 .popup,
 .cms-admin-modal {
+    &.app-aldryn_bootstrap3 {
+        .filerFile .filerClearer {
```
Comment (dhorelik) [inline djangocms_admin_style/sass/components/_popup.scss]: nesting doesn't seem necessary here: `&.app-aldryn_bootstrap3 .filerFile .filerClearer {}`

File: djangocms_admin_style/sass/components/_popup.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/components/_popup.scss]: yes I really avoid If it's possible

**Comment by Flight** [comment]
LGTM

**Pull request merged**